bpf/process/bpf_execve_event.c

```diff
@@ -185,10 +188,37 @@ event_execve(struct sched_execve_args *ctx)
 	fileoff = ctx->filename & 0xFFFF;
 	binary = event_filename_builder(ctx, execve, pid, EVENT_EXECVE, binary,
 					(char *)ctx + fileoff);
+	event->binary = binary;
```

Comment:
this reads a bit strange I'm not sure why

event->binary = event_filename_builder(...)

is not used.

Reply:
ugh, will fix that in another PR, thanks